app/controllers/shows_controller.rb

```diff
@@ -0,0 +1,8 @@
+class ShowsController < ApplicationController
+  include ShowsHelper
+
+  def index
+    @shows = parsed_csv_data.take(6)
```

Comment:
I like this `take` method. I'm interested as to why you extracted the thingy out to a helper!

Reply:
I thought the parsed_csv_data method is likely to be re-used? A bit premature maybe! Also I thought to keep the controller skinny, and I have no Show model yet to extract the logic of taking the first 6 items..?

Reply:
Reasonable! You'll see in the next challenge it might be a bit premature. But it's not a blocker!